babyagi.py

```diff
@@ -52,20 +64,28 @@
     os.getenv("ENABLE_COMMAND_LINE_ARGS", "false").lower() == "true"
 )
 if ENABLE_COMMAND_LINE_ARGS:
-    from extensions.argparseext import parse_arguments
+    if can_import("extensions.argparseext"):
+        from extensions.argparseext import parse_arguments
 
-    OBJECTIVE, INITIAL_TASK, OPENAI_API_MODEL, DOTENV_EXTENSIONS = parse_arguments()
+        OBJECTIVE, INITIAL_TASK, OPENAI_API_MODEL, DOTENV_EXTENSIONS = parse_arguments()
 
 # Load additional environment variables for enabled extensions
 if DOTENV_EXTENSIONS:
-    from extensions.dotenvext import load_dotenv_extensions
+    if can_import("extensions.dotenvext"):
+        from extensions.dotenvext import load_dotenv_extensions
 
-    load_dotenv_extensions(DOTENV_EXTENSIONS)
+        load_dotenv_extensions(DOTENV_EXTENSIONS)
 
 # TODO: There's still work to be done here to enable people to get
 # defaults from dotenv extensions # but also provide command line
 # arguments to override them
 
+# Extensions support end
+
+# Check if we know what we are doing
+assert OBJECTIVE, "OBJECTIVE environment variable is missing from .env"
```

Comment:
OBJECTIVE could also be loaded from command args right?
Seems like it's more correct to say is missing from .env or arg (if enabled)

Reply:
There's a bit of duplication but that is also covered in the argparseext. This is to cover the case where the command line is not enabled